common/changes/@microsoft/rush/lockfile-diff_2021-11-30-23-57.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "When computing the set of projects affected by a Git change, if the change contains the lockfile and the package manager is pnpm, diff the content of the lockfile to determine which projects were affeted by the lockfile change, instead of assuming all projects are affected.",
```

Comment:
```suggestion
      "comment": "When computing the set of projects affected by a Git change, if the change includes changes to the lockfile and the package manager is pnpm, diff the content of the lockfile to determine which projects were affected by the lockfile change, instead of assuming all projects are affected.",
```